mix.exs

```diff
@@ -24,20 +24,31 @@ defmodule MongoosePush.Mixfile do
 
   defp deps do
     [
-     {:pigeon, git: "https://github.com/rslota/pigeon.git", tag: "6d1e4e3"},
-     {:maru, git: "https://github.com/rslota/maru.git", tag: "7c1da75", override: true},
+     {:pigeon, github: "rslota/pigeon", tag: "087bb38"},
+     {:maru, github: "elixir-maru/maru", tag: "7a24d1a3", override: true},
      {:poison, "~> 3.0"},
-     {:httpoison, "~> 0.10.0"},
+     {:httpoison, "~> 0.11.0"},
      {:maru_swagger, github: "elixir-maru/maru_swagger"},
      {:distillery, "~> 1.0"},
      {:confex, "~> 1.4", override: true},
      {:mix_docker, "~> 0.3"},
+
+     # Just overrides to make elixometer compile...
+     {:setup, github: "uwiger/setup", tag: "1.8.0", override: true, manager: :rebar},
+     {:edown, github: "uwiger/edown", tag: "0.8", override: true},
+     {:lager, ">= 3.2.1", override: true},
+     {:exometer_core, github: "PSPDFKit-labs/exometer_core", override: true},
```

Comment:
This is a version with stripped down dependancies. The original one uses lots of deps that conflict with `exometer`. It's not pretty, but the author of exometer made a lot of mess while splitting `exometer` into two repos. I've tried several approaches to make it work and this seems to be the simples one. Also this was suggested by author of the `elixometer`